testing/openimageio/APKBUILD

```diff
@@ -0,0 +1,67 @@
+# Contributor: Mark Riedesel <mark@klowner.com>
+# Maintainer:
+pkgname=openimageio
+pkgver=1.6.14
+pkgrel=0
+pkgdesc="Image i/o library supporting a multitude of image formats"
+url="https://sites.google.com/site/openimageio/"
+arch="all"
+license="BSD"
+depends=""
+makedepends="cmake giflib-dev zlib-dev boost-dev jpeg-dev libpng-dev tiff-dev
+	glew-dev python2-dev openexr-dev ffmpeg2.8-dev libraw-dev libwebp-dev freetype-dev
+	qt-dev mesa-dev openssl-dev jasper-dev opencolorio-dev"
+install=""
+subpackages="$pkgname-python2:py2 $pkgname-python3:py3 $pkgname-dev $pkgname-doc $pkgname-utils"
```

Comment:
You can also use common split function `_python` for both of these subpackages. Then you can use `$subpkgname` in the split function to get name of the specific subpackage and e.g. `${subpkgname:2:1}` to extract `2` or `3`.